src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard2.0;netstandard2.1</TargetFrameworks>

```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/issues/716 - If possible add net452 too. 

cc: @eddynaka who worked on adding new framework support.

Reply:
Also, we need to add the same targets to .Tests project as well.

Reply:
@cijothomas,  I had some problems adding 452 because of the compatibility. Don't know yet what would be the best approach for those. I added my comments in the 716 issue to check it out what would be the possibilities. Let me know if you have an idea how could we approach that.

Reply:
>If possible add net452 too

I'm sorry, don't have much time to work around compatibility issues mentioned by @eddynaka. 

>Also, we need to add the same targets to .Tests project as well.

Done!